Source/hydro/advection_util.cpp

```diff
@@ -134,7 +142,10 @@ Castro::shock(const Box& bx,
 
     // now compute (grad P - rho g) . dx
     // We subtract off the hydrostatic force, since the pressure that
-    // balances that is not available to make a shock.
+    // balances that is not available to make a shock.  We compute this
+    // as:
+    // P'_{i+1} = P_{i+1} - [ P_i - \int_{x_i}^{x_{i+1}} rho g dx ]
```

Comment:
wouldn't that make it `+rho g dx` since there are two minus?

Reply:
I've made the comment clearer and fixed the sign